crates/node-core/src/args/rpc_server_args.rs

```diff
@@ -176,6 +176,14 @@ pub struct RpcServerArgs {
     /// Gas price oracle configuration.
     #[command(flatten)]
     pub gas_price_oracle: GasPriceOracleArgs,
+
+    /// Enable auth engine api over IPC  
+    #[arg(long, default_value_if("dev", "true", Some("true")))]
```

Comment:
I don't want to set this to default ever
```suggestion
    #[arg(long))]
```